### PR TITLE
Update migrate-to-microsoft-365-business.md

### PR DESCRIPTION
**Pull request by quietlikeninja**
Just a minor change from using "he or she" to "they"

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/business/migrate-to-microsoft-365-business.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ If you already have an Office 365 for business subscription, for example, Office
   
 ## Add Microsoft 365 Business licenses
 
-You have two ways to get Microsoft 365 Business. If you have a partner, he or she can purchase Microsoft 365 Business for you from [Microsoft Partner Center](get-microsoft-365-business.md). Your partner can also help you transition to Microsoft 365 Business.
+You have two ways to get Microsoft 365 Business. If you have a partner, they can purchase Microsoft 365 Business for you from [Microsoft Partner Center](get-microsoft-365-business.md). Your partner can also help you transition to Microsoft 365 Business.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/migrate-to-microsoft-365-business.md]:
```suggestion
You have two ways to get Microsoft 365 Business. A partner can purchase Microsoft 365 Business for you from [Microsoft Partner Center](get-microsoft-365-business.md). Your partner can also help you transition to Microsoft 365 Business.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@quietlikeninja - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. 

Thank you.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. Kindly assist to check and merge.

Thank you.

**Pull request merged**